src/actions/user/signupAction.spec.js

```diff
@@ -0,0 +1,70 @@
+import ReduxThunk from 'redux-thunk';
+import configureStore from 'redux-mock-store';
+import moxios from 'moxios';
+import signupAction from './signupAction';
+import {
+  USER_SIGNUP_REQUEST,
+  USER_SIGNUP_SUCCESS,
+  USER_SIGNUP_FAILURE,
+} from '../../actionTypes';
+
+const MOCK_USER = {
```

Comment:
can create a single source for our mocked resources, lets make our app DRY?

Reply:
@dinobi I have created a `mockData.js` file in the `helpers/` directory. I did it this way to conform with what Adeola already implemented in his PR.